.github/workflows/all-post-commit-workflows.yaml

```diff
@@ -6,6 +6,12 @@ on:
   push:
     branches: ["main", "fd-2/main"]
 
+permissions:
```

Comment:
Are these permissions ok to have on `Post commit` workflow?

Reply:
I think pages is unavoidable...

We do need `id-token` unfortunately: https://github.com/actions/deploy-pages?tab=readme-ov-file#usage